### PR TITLE
🌐 [i18n-KO] Translated `reference/inference_endpoints.md` to Korean

### PR DESCRIPTION
**Pull request by harheem**
# What does this PR do?

Translated the `reference/inference_endpoints.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team Hugging Face KREW, may you please review this PR? 
@harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae, @usr-bin-ksh, @fabxoe

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: docs/source/ko/package_reference/inference_endpoints.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# 추론 엔드포인트 [[inference-endpoints]]
+
+Hugging Face가 관리하는 추론 엔드포인트는 우리가 모델을 쉽고 안전하게 배포할 수 있게 해주는 도구입니다. 이러한 추론 엔드포인트는 [Hub](https://huggingface.co/models)에 있는 모델을 기반으로 설계되었습니다. 이 문서는 `huggingface_hub`와 추론 엔드포인트 통합에 관한 기술적 자료로써, 보다 자세한 정보는 [공식 문서](https://huggingface.co/docs/inference-endpoints/index)를 통해 확인할 수 있습니다.
+
+<Tip>
+
+'huggingface_hub'를 사용하여 추론 엔드포인트를 프로그래밍 방식으로 관리하는 방법을 알고 싶다면, [관련 가이드](../guides/inference_endpoints)를 확인해보세요.
```
Comment (harheem) [inline docs/source/ko/package_reference/inference_endpoints.md:7]:
```suggestion
'huggingface_hub'를 사용하여 추론 엔드포인트를 프로그래밍 방식으로 관리하는 방법을 알고 싶다면, [관련 가이드](../guides/inference_endpoints)를 확인해 보세요.
```

File: docs/source/ko/package_reference/inference_endpoints.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+</Tip>
+
+추론 엔드포인트는 API로 쉽게 접근할 수 있습니다. 이 엔드포인트들은 [Swagger](https://api.endpoints.huggingface.cloud/)를 통해 문서화되어 있고, [`InferenceEndpoint`] 클래스는 이 API를 사용해 만든 간단한 래퍼입니다.
+
+## 메서드 [[methods]]
```
Comment (harheem) [inline docs/source/ko/package_reference/inference_endpoints.md:13]:
```suggestion
## 매소드 [[methods]]
```

File: docs/source/ko/package_reference/inference_endpoints.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# 추론 엔드포인트 [[inference-endpoints]]
+
+Hugging Face가 관리하는 추론 엔드포인트는 우리가 모델을 쉽고 안전하게 배포할 수 있게 해주는 도구입니다. 이러한 추론 엔드포인트는 [Hub](https://huggingface.co/models)에 있는 모델을 기반으로 설계되었습니다. 이 문서는 `huggingface_hub`와 추론 엔드포인트 통합에 관한 기술적 자료로써, 보다 자세한 정보는 [공식 문서](https://huggingface.co/docs/inference-endpoints/index)를 통해 확인할 수 있습니다.
```
Comment (harheem) [inline docs/source/ko/package_reference/inference_endpoints.md:3]:
```suggestion
Hugging Face가 관리하는 추론 엔드포인트는 우리가 모델을 쉽고 안전하게 배포할 수 있게 해주는 도구입니다. 이러한 추론 엔드포인트는 [Hub](https://huggingface.co/models)에 있는 모델을 기반으로 설계되었습니다. 이 문서는 `huggingface_hub`와 추론 엔드포인트 통합에 관한 기술적 자료로써, 더욱 자세한 정보는 [공식 문서](https://huggingface.co/docs/inference-endpoints/index)를 통해 확인할 수 있습니다.
```

File: docs/source/ko/package_reference/inference_endpoints.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+다음과 같은 추론 엔드포인트의 기능이 [`HfApi`]안에 구현되어 있습니다:
+
+- [`get_inference_endpoint`]와 [`list_inference_endpoints`]를 사용해 엔드포인트 정보를 조회할 수 있습니다.
+- [`create_inference_endpoint`], [`update_inference_endpoint`], [`delete_inference_endpoint`]로 엔드포인트를 배포하고 관리할 수 있습니다.
+- [`pause_inference_endpoint`]와 [`resume_inference_endpoint`]로 엔드포인트를 잠시 멈추거나 다시 시작할 수 있습니다.
+- [`scale_to_zero_inference_endpoint`]로 엔드포인트의 복제본을 0개로 설정할 수 있습니다.
+
+## InferenceEndpoint [[huggingface_hub.InferenceEndpoint]]
+
+[`InferenceEndpoint`]는 배포된 `InferenceEndpoint`에 대한 정보를 가진 주 데이터 클래스입니다. 여기에는 구성과 현재 상태가 포함되어 있습니다. 배포 후에는 [`InferenceEndpoint.client`]와 [`InferenceEndpoint.async_client`]를 사용해 엔드포인트에서 추론 작업을 할 수 있고, 이때 [`InferenceClient`]와 [`AsyncInferenceClient`] 객체를 반환합니다.
```
Comment (seoulsky-field) [inline docs/source/ko/package_reference/inference_endpoints.md:24]:
```suggestion
기본 데이터 클래스는 [`InferenceEndpoint`]입니다. 여기에는 구성 및 현재 상태를 가지고 있는 배포된 `InferenceEndpoint`에 대한 정보가 포함되어 있습니다. 배포 후에는 [`InferenceEndpoint.client`]와 [`InferenceEndpoint.async_client`]를 사용해 엔드포인트에서 추론 작업을 할 수 있고, 이때 [`InferenceClient`]와 [`AsyncInferenceClient`] 객체를 반환합니다.
```

원문 문장 구성과 조금 다르게 구성되어 있어 살짝 수정해보았습니다!

**Review by seoulsky-field — COMMENTED** [review]
좋은 번역 감사합니다! 원문과 문장 구성이 살짝 다른 부분이 있어서 수정해보았습니다!

File: docs/source/ko/package_reference/inference_endpoints.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# 추론 엔드포인트 [[inference-endpoints]]
+
+Hugging Face가 관리하는 추론 엔드포인트는 우리가 모델을 쉽고 안전하게 배포할 수 있게 해주는 도구입니다. 이러한 추론 엔드포인트는 [Hub](https://huggingface.co/models)에 있는 모델을 기반으로 설계되었습니다. 이 문서는 `huggingface_hub`와 추론 엔드포인트 통합에 관한 기술적 자료로써, 더욱 자세한 정보는 [공식 문서](https://huggingface.co/docs/inference-endpoints/index)를 통해 확인할 수 있습니다.
```
Comment (junejae) [inline docs/source/ko/package_reference/inference_endpoints.md:3]:
```suggestion
Hugging Face가 관리하는 추론 엔드포인트는 우리가 모델을 쉽고 안전하게 배포할 수 있게 해주는 도구입니다. 이러한 추론 엔드포인트는 [Hub](https://huggingface.co/models)에 있는 모델을 기반으로 설계되었습니다. 이 문서는 `huggingface_hub`와 추론 엔드포인트 통합에 관한 참조 페이지이며, 더욱 자세한 정보는 [공식 문서](https://huggingface.co/docs/inference-endpoints/index)를 통해 확인할 수 있습니다.
```

Comment (junejae) [reply docs/source/ko/package_reference/inference_endpoints.md:3]: This page is a reference for huggingface_hub’s integration with Inference Endpoints.

원문이 위와 같아서 좀 더 자연스럽게 수정해봤습니다.

**Review by Wauplin — APPROVED** [review]
Awesome! Thanks again!

**Pull request merged**